src/main/java/ch/saunah/saunahbackend/SaunahBackendApplication.java

```diff
@@ -102,6 +102,19 @@ protected void configure(HttpSecurity http) throws Exception {
                 "/allBookings",
                 "/bookings/{id}/approve"
             ).hasAuthority(UserRole.ADMIN.toString())
+            //users
+            .mvcMatchers(
+                HttpMethod.GET,
+                "/users/{id}"
+            ).hasAnyAuthority(UserRole.USER.toString(), UserRole.ADMIN.toString())
+            .antMatchers(
+                "/users",
+                "users/userRole/{id}"
```

Comment:
I would suggest that we change the route to `usrs/user-role/{id}`, as we used notation with dashes (instead of camelCase) in other places already, and I think it's nice to be consistent with this throughout the whole application.

Reply:
userRole is done over editUser now. Therefore i removed this autohrization